server/ethereum/ethereum.test.js

```diff
@@ -29,18 +30,87 @@ afterAll(() => {
   web3.currentProvider.connection.close()
 })
 
-// NOTE - this isn't really testing anything yet, just showing how to fire an event and getting the testing framework working
-test('firing an event pins a file', async done => {
+test('watcher pins file from registerData function', async done => {
   const testKey = web3.utils.fromAscii('testKey')
-  const testVal = { dag: 'killinnnnnn it' }
-  const hash = await node.dag.put(testVal)
+  const hash = await node.dag.put(dag)
 
   registerWatcher(contract)
   contract.methods
     .registerData(testKey, hash.toBaseEncodedString())
     .send({ from: accounts[0] }, () => {
-      setTimeout(() => {
+      setTimeout(async () => {
```

Comment:
what does this do? looks like it just manually pins event and then checks that it pinned? confused how this shows the watcher is pinning it